docs/source/onnx.rst

```diff
@@ -320,16 +570,90 @@ Now PyTorch is able to export ``elu`` operator.
+you can create and register your own custom ops implementation in PyTorch. Here's how to export such model to ONNX.::
+
+    # Create custom symbolic function
+    from torch.onnx.symbolic_helper import parse_args
+    @parse_args('v', 'v', 'f', 'i')
+    def symbolic_foo_forward(g, input1, input2, attr1, attr2):
+        return g.op("Foo", input1, input2, attr1_f=attr1, attr2_i=attr2)
+
+    # Register custom symbolic function
+    from torch.onnx import register_custom_op_symbolic
+    register_custom_op_symbolic('custom_ops::foo_forward', symbolic_foo_forward, 9)
+
+    class FooModel(torch.nn.Module):
+        def __init__(self, attr1, attr2):
+            super(FooModule, self).__init__()
+            self.attr1 = attr1
+            self.attr2 = attr2
+
+        def forward(self, input1, input2):
+            # Calling custom op
+            return torch.ops.custom_ops.foo_forward(input1, input2, self.attr1, self.attr2)
+
+    model = FooModel(attr1, attr2)
+    torch.onnx.export(model, (dummy_input1, dummy_input2), 'model.onnx')
+
+Depending on the custom operator, you can export it as one or a combination of existing ONNX ops.
+You can also export it as a custom op in ONNX as well. In that case, you will need to extend the backend of your choice
+with matching custom ops implementation, e.g. `Caffe2 custom ops <https://caffe2.ai/docs/custom-operators.html>`_,
+`ONNXRuntime custom ops <https://github.com/microsoft/onnxruntime/blob/master/docs/AddingCustomOp.md>`_.
+
+Frequently Asked Questions
+--------------------------
+Q: I have exported my lstm model, but its input size seems to be fixed?
+
+  The tracer records the example inputs shape in the graph. In case the model should accept
+  inputs of dynamic shape, you can utilize the parameter `dynamic_axes` in export api. ::
+
+    layer_count = 4
+
+    model = nn.LSTM(10, 20, num_layers=layer_count, bidirectional=True)
+    model.eval()
+
+    with torch.no_grad():
+        input = torch.randn(5, 3, 10)
+        h0 = torch.randn(layer_count * 2, 3, 20)
+        c0 = torch.randn(layer_count * 2, 3, 20)
+        output, (hn, cn) = model(input, (h0, c0))
+
+        # default export
+        torch.onnx.export(model, (input, (h0, c0)), 'lstm.onnx')
+        onnx_model = onnx.load('lstm.onnx')
+        # input shape [5, 3, 10]
+        print(onnx_model.graph.input[0])
+
+        # export with `dynamic_axes`
+        torch.onnx.export(model, (input, (h0, c0)), 'lstm.onnx',
+                        input_names=['input', 'h0', 'c0'],
+                        output_names=['output', 'hn', 'cn'],
+                        dynamic_axes={'input': {0: 'sequence'}, 'output': {0: 'sequence'}})
+        onnx_model = onnx.load('lstm.onnx')
+        # input shape ['sequence', 3, 10]
+        print(onnx_model.graph.input[0])
+
+
+Q: How to export models with loops in it?
+
+  Please checkout `Tracing vs Scripting`_.
+
 Functions
 --------------------------
 .. autofunction:: export
+.. autofunction:: _export
```

Comment:
Is there any benefit to expose `_export`? It's pretty much the same as export, except some non-official parameters which may lead to non-standard onnx export. If we really want to do so, let's emphasis that, users are on their own risks :-)

Reply:
you are right, i don't see much benefit. Most of the useful parameters are already added to export. I'm removing this.